examples/egl_common.h

```diff
@@ -0,0 +1,19 @@
+#ifndef _EGL_COMMON_H
+#define _EGL_COMMON_H
+#endif
+
+#include <stdbool.h>
+#include <wayland-client.h>
+
+#include <EGL/egl.h>
+#include <EGL/eglext.h>
+
+extern EGLDisplay egl_display;
+extern EGLConfig egl_config;
+extern EGLContext egl_context;
+
+extern PFNEGLCREATEPLATFORMWINDOWSURFACEEXTPROC eglCreatePlatformWindowSurfaceEXT;
+
+bool egl_init(struct wl_display *display, const EGLint *config_attribs);
```

Comment:
OK, thanks for the update, looks almost ready for merging. One last comment: can we get rid of this `config_attribs` param? No example needs a special EGL config. This config should be more correct than the empty config and should be fine with all examples:

```c
EGLint config_attribs[] = {
	EGL_SURFACE_TYPE, EGL_WINDOW_BIT,
	EGL_RED_SIZE, 1,
	EGL_GREEN_SIZE, 1,
	EGL_BLUE_SIZE, 1,
	EGL_ALPHA_SIZE, 1,
	EGL_RENDERABLE_TYPE, EGL_OPENGL_ES2_BIT,
	EGL_NONE,
};
```

Reply:
Done! Hopefully, these `config_attribs` will work with all of the affected example clients.